packages/console/src/pages/ConnectorDetails/index.module.scss

```diff
@@ -64,6 +64,10 @@
       @include _.vertical-bar;
       height: 12px;
     }
+
+    .copy {
+      padding: _.unit(1) _.unit(2);
```

Comment:
Should we add the padding in `CopyToClipboard` component? How about application ID, user ID in the other details page?

Reply:
Good call, previously, they don't follow the same design.

Updated, PTAL